### PR TITLE
Updating CI env with 2022.04 qa-niaid.planx-pla.net 1649349213

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2022.04 to qa-niaid.planx-pla.net

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-niaid.planx-pla.net/manifest.json
## :warning: Services on branch
- fence
## Deployment changes
- wts
  - This version of WTS requires a DB migration (in cloud-automation deployments, the DB should be migrated automatically) (uc-cdis/workspace-token-service#40)

**Comment by PlanXCyborg** [comment]
# qa-niaid.planx-pla.net/manifest.json
## :warning: Services on branch
- fence
## Deployment changes
- wts
  - This version of WTS requires a DB migration (in cloud-automation deployments, the DB should be migrated automatically) (uc-cdis/workspace-token-service#40)

File: qa-niaid.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -7,29 +7,29 @@
...
   },
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2022.03",
-    "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:2022.03",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2022.04",
+    "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:2022.04",
     "aws-es-proxy": "quay.io/cdis/aws-es-proxy:0.8",
-    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2022.03",
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2022.04",
     "fence": "quay.io/cdis/fence:debug_immport",
```
Comment (paulineribeyre) [inline qa-niaid.planx-pla.net/manifest.json:14]: you'll need to deploy fence:2022.04 to test the release - lmk when you do 🙏

File: qa-niaid.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -7,29 +7,29 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2022.03",
-    "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:2022.03",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2022.04",
+    "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:2022.04",
     "aws-es-proxy": "quay.io/cdis/aws-es-proxy:0.8",
```
Comment (paulineribeyre) [inline qa-niaid.planx-pla.net/manifest.json:12]: isn't this one on ECR too now?

File: qa-niaid.planx-pla.net/manifest.json [hunk 3]
```diff
@@ -7,29 +7,29 @@
...
+    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2022.04",
+    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2022.04",
     "ambassador": "quay.io/datawire/ambassador:1.4.2",
-    "guppy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/guppy:2022.03",
-    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2022.03",
-    "manifestservice": "quay.io/cdis/manifestservice:2022.03",
-    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2022.03"
+    "guppy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/guppy:2022.04",
+    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2022.04",
+    "manifestservice": "quay.io/cdis/manifestservice:2022.04",
```
Comment (paulineribeyre) [inline qa-niaid.planx-pla.net/manifest.json:31]: quay

**Comment by PlanXCyborg** [comment]
# qa-niaid.planx-pla.net/manifest.json
## Deployment changes
- wts
  - This version of WTS requires a DB migration (in cloud-automation deployments, the DB should be migrated automatically) (uc-cdis/workspace-token-service#40)

**Comment by PlanXCyborg** [comment]
# qa-niaid.planx-pla.net/manifest.json
## Deployment changes
- wts
  - This version of WTS requires a DB migration (in cloud-automation deployments, the DB should be migrated automatically) (uc-cdis/workspace-token-service#40)

**Comment by PlanXCyborg** [comment]
# qa-niaid.planx-pla.net/manifest.json
## :warning: Services on branch
- fence
## Deployment changes
- wts
  - This version of WTS requires a DB migration (in cloud-automation deployments, the DB should be migrated automatically) (uc-cdis/workspace-token-service#40)

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**